src/main/java/com/locallampoon/fiveh/client/FiveHClient.java

```diff
@@ -9,6 +9,6 @@
 
     public static void main(String[] args) throws IOException {
         game = new Game();
-        game.start();
+        game.startV2();
```

Comment:
Non-blocking, but perhaps you could define or comment this method more descriptively, like so... 
`// V2 initializes the GUI` or `game.startWithGUI`
Just a suggestion.